retreaver-typescript/src/base/Model.ts

```
@@ -0,0 +1,188 @@
+      Model.updateVisitorId(data);
+    }
+    return this._store;
+  }
+
+  /**
+   * Get data from API
+   * @param path - API path
+   * @param callback - Callback function
+   * @returns Request result
+   */
+  protected getData(path: string, callback?: Function): any {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)